bulletin-buddy-offer-service/src/main/java/com/github/bulletinboard/services/OfferService.java

```diff
@@ -21,4 +22,13 @@ public void removeAllOffers() {
     public void addOffer(Offer offer) {
         offers.add(offer);
     }
+
+    public Offer getOfferById(UUID id) {
```

Comment:
I think this could be optimized using `stream` instead of iterating and checking, but since we will have the database we can query with the Id, for now this fine

Reply:
Alright. Thanks for your time!